hazelcast/src/main/java/com/hazelcast/internal/partition/IPartitionService.java

```diff
@@ -163,6 +163,8 @@ public interface IPartitionService extends CoreService {
      */
     boolean isMemberStateSafe();
 
+    boolean isPartitionTableHealthy();
```

Comment:
maybe javadoc what the difference with `isMemberStateSafe()` is.

Reply:
done in `a749decb`